.gitignore

```diff
@@ -33,6 +33,8 @@
 
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
+.vscode/
```

Comment:
Please revert these changes as they are not related to this issue. If you think that we have to add these files into ignore list could you please make another issue for this? It's easy to track changes if they are separate :)